CONTRIBUTING.md

```diff
@@ -38,6 +38,39 @@ For example, for running tests in `multi`, `uuid` and `main` packages, call
 make test-multi test-uuid test-main
 ```
 
+Before running benchmark or measuring performance degradation, install benchmark dependencies:
+```bash
+make bench-deps BENCH_PATH=custom_path
+```
+
+Use the variable `BENCH_PATH` to specialize the path of benchmark artifacts.
+It is set to `bench` by default.
+
+To run benchmark tests, call:
+```bash
+make bench DURATION=5s COUNT=7 BENCH_PATH=custom_path
+```
+
+Use the variable `DURATION` to set the duration of perf tests. That variable is mapped on
+testing [flag](https://pkg.go.dev/cmd/go#hdr-Testing_flags) `-benchtime` for gotest.
+It may take the values in seconds (e.g, `5s`) or count of iterations (e.g, `1000x`).
+It is set to `3s` by default.
+
+Use the variable `COUNT` to control the count of benchmark runs for each test. It is set to `5` by default.
+That variable is mapped on testing flag `-count`.
+Use higher values if the benchmark numbers aren't stable.
+
+Before comparing performance difference between master and the current branch, 
+prepare benchmark results of master:
+```bash
+make perf-reference DURATION=5s COUNT=7 BENCH_PATH=custom_path
+```
+
+For need of measuring performance degradation after changes in code, run perf-diff:
+```bash
+make perf-diff BENCH_PATH=custom_path
```

Comment:
It's not obvious here that we must have `make bench` results here. I thought that make perf-diff runs it one more time, as in previous patch. Maybe we should add a comment.

Reply:
Fixed.